discord_thread_summarizer_using_openai/discord_utils.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+import ballerina/time;
+
+const DISCORD_API_URL = "https://discord.com/api/v9";
+const time:Seconds DISCORD_EPOCH = 1420070400000.00;
+
+configurable string botToken = ?;
+
+final map<string> & readonly headers = {
+    "Authorization": botToken
+};
```

Comment:
```suggestion
};

```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/9018e16abefb68f7d46cfbc2a86a00240ce6ea5f